app/views/incarceration.xml

```diff
@@ -0,0 +1,29 @@
+<Alloy>		
+		<Window class="container">
+			<View id="header">
+				<ImageView id="headerLeft" image="/global/blind2.png"></ImageView>
+				<Label id="headerText">Victim Services</Label>
+				<ImageView id="headerRight" image="/global/add126.png"></ImageView>
+			</View>
+			<TableView>
```

Comment:
More tech debt than something to change right now, but we should try to only use TableViews when we are actually listing data as opposed to using them when we are trying to achieve a vertical layout. Items can still be laid out vertically by putting those items inside of a View tag and then setting the view to have a vertical layout.
Example:
```
<View layout="vertical">
    <Label>Hello, World!</Label>
    <Label>hello world is sitting on top of me</Label>
</View>